### PR TITLE
Front end form reuse

### PR DESCRIPTION
**Pull request by tessathornberry**
#61

### CODE REVIEW

File: src/components/case_manager/StudentParaForm.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+import React from "react";
+import styles from "../../styles/Dashboard.module.css";
+
+interface Props {
+  title: string;
+  handleSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
+}
+
+const StudentParaForm = ({ title, handleSubmit }: Props) => {
```
Comment (codetheweb) [inline src/components/case_manager/StudentParaForm.tsx:12]: 🐧 maybe 
```suggestion
const PersonCreationForm = ({ title, handleSubmit }: Props) => {
```
?

File: src/components/case_manager/StudentParaForm.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/* This is a functional <form> component to be imported by both the AllStudents.tsx and the AllParas.tsx
+NOTE: Subsequent list rendering and trpc calls would be challenging to reuse, but this form element can be used successfully in the front end for users to create or add students or paras to their caseloads, populating both lists. */
+
+import React from "react";
+import styles from "../../styles/Dashboard.module.css";
+
+interface Props {
+  title: string;
+  handleSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
```
Comment (codetheweb) [inline src/components/case_manager/StudentParaForm.tsx:9]: the convention in React is to usually have event handler in props be named something like `on...`, so maybe

```suggestion
  onSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
```

?

File: src/components/case_manager/AllStudents.tsx [hunk 3]
```diff
@@ -25,35 +26,10 @@ const AllStudentsPage = () => {
     return <div>Loading...</div>;
   }
 
+  // reusable Functional Component for form only
```
Comment (codetheweb) [inline src/components/case_manager/AllStudents.tsx:29]:
```suggestion
```

File: src/components/case_manager/AllParas.tsx [hunk 4]
```diff
@@ -26,30 +27,10 @@ const AllParasPage = () => {
     return <div>Loading...</div>;
   }
 
+  // reusable Functional component for form only
```
Comment (codetheweb) [inline src/components/case_manager/AllParas.tsx:30]:
```suggestion
```

**Review by codetheweb — COMMENTED** [review]
looks good!

not sure if I mentioned this before but I use 🐧 to indicate small changes/preferences that don't necessarily need to be resolved before merge depending on how you feel about my suggestion

File: src/components/case_manager/AllParas.tsx (hunk 4 above)
Comment (tessathornberry) [reply src/components/case_manager/AllParas.tsx]: Thank you, @codetheweb ! I made the changes and pushed my edits up!

**Review by codetheweb — APPROVED** [review]

**Pull request merged**